roles/openshift_excluder/tasks/install.yml

```diff
@@ -2,15 +2,15 @@
 - name: Install latest excluder
   package:
     name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
+    state: "{{ excluder_package_state }}"
   when:
   - openshift_excluder_enabled | default(false) | bool
   - not openshift.common.is_containerized | bool
 
 - name: Install latest docker excluder
   package:
     name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
+    state: "{{ docker_excluder_package_state }}"
   when:
   - docker_excluder_enabled | default(false) | bool
   - not openshift.common.is_containerized | bool
```

Comment:
I have introduce another playbook ``playbooks/common/openshift-cluster/upgrades/disable_excluder.yml`` that is used in upgrades only instead of the ``playbooks/common/openshift-cluster/disable_excluder.yml``